contracts/AelinPool.sol

```diff
@@ -93,17 +97,19 @@ contract AelinPool is AelinERC20, MinimalProxyFactory {
         aelinDealLogicAddress = _aelinDealLogicAddress;
         aelinRewardsAddress = _aelinRewardsAddress;
 
-        emit SetSponsor(_sponsor);
-    }
-
-    function updateAllowList(address[] memory _allowList, uint256[] memory _allowListAmounts) external onlyPoolFactoryOnce {
```

Comment:
@alextheboredape I removed this function since it's not being used anymore. I assume we don't need the checks maded in `onlyPoolFactoryOnce` since everything happens in the `init` function already?
Like we don't need  anything in `require(msg.sender == poolFactory && !hasAllowList && totalSupply() == 0, "only pool factory can access");` imo but please double check this